src/Codegen/Generator.hack

```diff
@@ -119,24 +119,28 @@ class Query extends BaseObject<QueryField> {
     }
 }
 
-class Object extends BaseObject<Field> {
+abstract class BaseClassishType<T as \Slack\GraphQL\__Private\Classish> extends BaseObject<Field> {
```

Comment:
There has to be a better name for this class 😓

Reply:
The GraphQL spec calls objects, interfaces and unions "composite types".

Reply:
I like that. Will rename.